pkg/datapath/linux/routing/routing_test.go

```diff
@@ -43,7 +43,22 @@ func TestPrivilegedConfigure(t *testing.T) {
 
 	ns2 := netns.NewNetNS(t)
 	ns2.Do(func() error {
-		ip, ri := getFakes(t, false, false)
+		ip, ri := getFakes(t, ipamOption.IPAMAzure, false, false)
+		masterMAC := ri.MasterIfMAC
+		ifaceCleanup := createDummyDevice(t, masterMAC)
+		defer ifaceCleanup()
+
+		runConfigureThenDelete(t, ri, ip, 1500)
```

Comment:
curious, what this test validates? does it check if real netlink api work as expected and program routes/rules?